Source/Csla.AspNetCore/ConfigurationExtensions.cs

```diff
@@ -35,8 +35,11 @@ public static CslaOptions AddAspNetCore(this CslaOptions config)
     /// </summary>
     /// <param name="config">CslaOptions instance</param>
     /// <param name="options">Options object</param>
-    public static CslaOptions AddAspNetCore(this CslaOptions config, Action<AspNetCoreConfigurationOptions> options)
+    /// <exception cref="ArgumentNullException"><paramref name="config"/> is <see langword="null"/>.</exception>
+    public static CslaOptions AddAspNetCore(this CslaOptions config, Action<AspNetCoreConfigurationOptions>? options)
     {
+      ArgumentNullException.ThrowIfNull(config);
```

Comment:
Shouldn't this throw on `options` instead of `config`?

Reply:
No. Config is the instance we are working with and the `options` delegte is optional which is expressed with `null`. See line 44 and 30.